.github/workflows/build-and-push-pms-spec.yml

```
@@ -0,0 +1,56 @@
+name: Build and Push Plex Media Server API Spec
+
+permissions:
+  checks: write
+  contents: write
+  pull-requests: write
+  statuses: write
+
+on:
```

yamllint

[warning] truthy value should be one of [false, true]

(truthy)